pkg/util/iptables/iptables.go

```diff
@@ -210,6 +214,12 @@ func newInternal(exec utilexec.Interface, protocol Protocol, lockfilePath string
 		version = MinCheckVersion
 	}
 
+	supportRandomFullyByKernel, err := randomFullyKernelCheck()
+	if err != nil {
+		klog.Warningf("Error checking kernel version, assuming iptables --random-fully not support by current kernel")
+		supportRandomFullyByKernel = false
```

Comment:
I think it would make more sense to put the fallback into `randomFullyKernelCheck()` (and then you don't need a temporary variable here, you can just do `hasRandomFully:  version.AtLeast(RandomFullyMinVersion) && randomFullyKernelCheck()`.

Also, rename `randomFullyKernelCheck` to `kernelSupportsRandomFully` ?